schemes/rasch_kristjansson/rk_stratiform.F90

```diff
@@ -0,0 +1,571 @@
+! Copyright (C) 2025 University Corporation for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+!
+! Rasch and Kristjansson prognostic cloud microphysics and CAM4 macrophysics
+! CCPP-ized: Haipeng Lin, January 2025
+module rk_stratiform
+  use ccpp_kinds, only: kind_phys
+
+  implicit none
+  private
+  save
+
+  ! public CCPP-compliant subroutines
+  !   note: cloud_fraction_perturbation_run is NOT a fully CCPP-compliant scheme
+  !         because it calls the compute_cloud_fraction scheme for perturbation.
+  !         this is currently not possible to elegantly handle in the framework.
+  !
+  ! refer to test SDF suite_rasch_kristjansson.xml for total order of operations,
+  ! as the full RK-stratiform requires other schemes not included in this module.
+  public :: rk_stratiform_init
+  public :: rk_stratiform_check_qtlcwat_run
+  ! -- cloud_particle_sedimentation --
+  public :: rk_stratiform_sedimentation_run
+  public :: rk_stratiform_detrain_convective_condensate_run
+  public :: rk_stratiform_cloud_fraction_perturbation_run         ! see note.
+  public :: rk_stratiform_external_forcings_run
+  public :: rk_stratiform_condensate_repartioning_run
+  ! -- prognostic_cloud_water --
+  public :: rk_stratiform_prognostic_cloud_water_tendencies_run
+  public :: rk_stratiform_cloud_optical_properties_run
+  public :: rk_stratiform_save_qtlcwat_run
+
+  !
+
+contains
+
+  ! Initialize rk_stratiform
+!> \section arg_table_rk_stratiform_init Argument Table
+!! \htmlinclude arg_table_rk_stratiform_init.html
+  subroutine rk_stratiform_init( &
+    errmsg, errflg)
+
+    ! Output arguments
+    character(len=512), intent(out)   :: errmsg         ! error message
+    integer,            intent(out)   :: errflg         ! error flag
+
+    errmsg = ''
+    errflg = 0
+
+  end subroutine rk_stratiform_init
```

Comment:
Is this subroutine actually needed?  I don't believe the CCPP has a requirement for an `init` phase.